pom.xml

```diff
@@ -477,11 +477,6 @@
         <surefire.environment>default</surefire.environment>
 
         <!-- Elasticsearch tests properties -->
-
-        <!-- Control how an Elasticsearch instance is run automatically during tests -->
-        <test.elasticsearch.run.skip>false</test.elasticsearch.run.skip>
```

Comment:
I think this one is still used? Let's keep it here?

Reply:
Yes it is, in order to not change too much the CI configuration. Right. Let's keep it.